### ISSUE TITLE
Scheduled Challenge "Next" Button Not Working

### ISSUE THREAD

**Issue by noslrac27** [SaitoTech/saito-lite-rust#1957]
Scheduled Challenge does not work since the "Next" button does nothing when being clicked. This is the "Next" button upon choosing the date and time.

<img width="960" alt="image" src="https://github.com/SaitoTech/saito-lite-rust/assets/101871246/88c0ceb0-a7b6-4449-a9e9-439ef8394f1d">

**Comment by f8daniel** [comment]
Yeah, we don't have the functionality yet. It's down the to-do list some ways
